internal/handlers/tigris/fetch.go

```diff
@@ -38,6 +41,14 @@ func (h *Handler) fetch(ctx context.Context, param fetchParam) ([]*types.Documen
 
 	collection, err := db.DescribeCollection(ctx, param.collection)
 	if err != nil {
+		if status.Code(err) == codes.NotFound {
```

Comment:
@AlekSi so, I stayed with gRPC for now, at least it doesn't introduce new dependencies. I'll do a small errors research in a separate PR.